hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BlockSender.java

```diff
@@ -631,6 +631,7 @@ private int sendPacket(ByteBuffer pkt, int maxChunks, OutputStream out,
          * 
          * Reporting of this case is done in DataXceiver#run
          */
+        LOG.warn("There was a timeout when sending packets: ", e);
```

Comment:
nit: `LOG.warn("Sending packets timed out.", e);`

Reply:
I am fine with the update though the exception would be threw and processed in DataXceiver#run.